.github/workflows/ci.yml

```diff
@@ -35,10 +35,15 @@ jobs:
         with:
           node-version-file: .nvmrc
           cache: pnpm
+      - name: Start Agents
+        run: npx nx-cloud start-ci-run --distribute-on=".nx/workflows/dynamic-changesets.yaml" --stop-agents-on-failure=false
       - name: Install dependencies
         run: pnpm install --frozen-lockfile --prefer-offline
       - name: Run Tests
         run: pnpm run test:ci
+      - name: Stop Agents
+        if: ${{ always() }}
+        run: npx nx-cloud stop-all-agents
       - name: Publish
```

Comment:
Hey @ZackDeRose , I just want to confirm that the main CI machine will have access to the build outputs from the Nx agents?

Reply:
Nvm I tested this myself and it worked :)